immutable/tsconfig.json

```diff
@@ -3,6 +3,7 @@
   "compilerOptions": {
     "outDir": "./dist",
     "rootDir": "..",
+    "importHelpers": true,
```

Comment:
ok looks like it's overriden in rollup ts plugin so guess this can be removed? I'll remove the one on bundler side

Reply:
I think it's still necessary to keep it here so we know it's enabled explicitly (for those who don't know about rollup plugins), and how we can disable it.